$NetBSD: patch-ac,v 1.3 2002/11/06 19:19:00 atatat Exp $

Control k already *has* a perfectly good meaning, so instead of
defeating that purpose, use control c to pop up the colour picker
instead.  I'm not sure exactly when the gaim people changed this
(somewhere between 0.59.1 and 0.59.3), but I think it's rather ill
advised.

--- src/conversation.c.orig	Sat Sep 14 10:13:05 2002
+++ src/conversation.c
@@ -955,8 +955,8 @@ gboolean keypress_callback(GtkWidget *en
 				toggle_font(c->font, c);
 				gtk_signal_emit_stop_by_name(GTK_OBJECT(entry), "key_press_event");
 				break;
-			case 'k':
-			case 'K':
+			case 'c':
+			case 'C':
 				quiet_set(c->fgcolorbtn,
 					  !gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(c->fgcolorbtn)));
 				toggle_fg_color(c->fgcolorbtn, c);
